docs/players-and-controllers/SLIMP3.md

```diff
@@ -3,12 +3,16 @@ layout: default
 title: SLIMP3
 ---
 
+# SLIMP3
+
+<figure markdown="span">
```

Comment:
May I ask what that instruction is for (besides the fact that it would center an image)?